neuraxle/base.py

```diff
@@ -1423,6 +1433,19 @@ def fit_transform(self, data_inputs, expected_outputs):
             step.set_train(is_train)
         return self
 
+    def __repr__(self):
+        main_step_output = super(TruncableSteps, self).__repr__()
+
+        output = "main step : " + '\n' \
+                 + main_step_output + '\n' \
+                 + "intermediate steps : " + '\n' \
+                 + pprint.pformat(self.steps_as_tuple) \
+                 + '\n'
```

Comment:
Same comment here, but with `(wrapped=..., hyperparams=...)`.

Reply:
Also implement the `__repr__` and `__str__` for the `TruncableSteps` class, which will call each of it's steps' `__repr__` method.

Reply:
`wrapped=self.wrapped.__repr__()`

Reply:
done